.github/workflows/post-release.yml

```
@@ -28,11 +28,11 @@ jobs:
           current_version=$(cargo metadata --format-version 1 --no-deps | \
             jq --raw-output '.packages | .[] | select(.name == "bevy").version')
           # Sanity check: current version should be 0.X.Y
-          if ! grep -q '^0\.[0-9]\+\.[0-9]\+$' <<< "${current_version}"; then
-            echo "Invalid version (not in 0.X.Y format): ${current_version}"
+          if ! grep -q '^0\.[0-9]\+\.[0-9]\+-dev$' <<< "${current_version}"; then
+            echo "Invalid version (not in 0.X.Y-dev format): ${current_version}"
```

```suggestion
          if ! grep -q '^0\.[0-9]\+\.[0-9]\+\-dev$' <<< "${current_version}"; then
            echo "Invalid version (not in 0.X.Y\-dev format): ${current_version}"
```
`-` formally indicates a range

---

not outside of `[..]`

---

Fair enough